pkgs/development/python-modules/langchain-tests/default.nix

```diff
@@ -23,14 +23,14 @@
 
 buildPythonPackage rec {
   pname = "langchain-tests";
-  version = "0.3.8";
+  version = "0.3.33";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "langchain-ai";
     repo = "langchain";
-    tag = "langchain-tests==${version}";
-    hash = "sha256-IZJo4EZFVKinBQdacM5xQ8ip3qTB64eqwZ9n+Z5mzWY=";
+    tag = "langchain-core==${version}";
```

Comment:
```suggestion
    tag = "langchain-tests==${version}";
```

Reply:
I can't believe I missed this.

Reply:
I don't have write access, so I can't fix it (sorry).

Reply:
Please open the new PR fixing this update behaviour

Reply:
Bug:  #379898